usecases/service_accounts_test.go

```diff
@@ -55,6 +55,7 @@ func TestServiceAccountsCreateShouldCreateRoleAndRoleBinding(t *testing.T) {
 }
 
 type saHasPermissionTestCase struct {
+	name     string
 	regPs    []string
```

Comment:
Please rename  regPs and test - I'm not sure what the former means and the latter is too generic.